app/lib/screens/history/history.dart

```diff
@@ -23,6 +27,26 @@ class _TravelHistoryState extends State<TravelHistory> {
           style: Theme.of(context).textTheme.headline5,
         ),
       ),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () async {
+          try {
+            buildPdf();
```

Comment:
And yes, as per Clark's prior comment please be aware this is to be revisited.

Reply:
> Just wondering about the file name change.
> 
> Additionally, you marked off a previous comment of mine about leaving the app with PDF open causing more RAM to be consumed. Did you happen to see this on your machine as well? What about you @MummenRider ?

I did not experienced any even I have 4GB RAM.

Reply:
OK, I imagine this is my personal problem then.